common/src/main/java/bisq/common/proto/persistable/PersistableListAsObservable.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.common.proto.persistable;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+
+import java.util.List;
+
+public abstract class PersistableListAsObservable<T extends PersistablePayload> extends PersistableList<T> {
+
+    public PersistableListAsObservable() {
+    }
+
+    protected PersistableListAsObservable(List<T> list) {
```

Comment:
Should take `ObservableList<T>`

Reply:
No, that is called from protobug methods and there we map to a list, will be copied over to the observable in the base class. We could use a collection here as well to make it more clear that its not setting the list.

Reply:
Right you are, since the list is created by `createList()`. No need to change, this is correct.

Reply:
See last commit...